src/app/components/Image/index.tsx

```diff
@@ -174,10 +222,21 @@ const Image = ({
               width={width}
               height={height}
               css={[
-                styles.image,
+                isPortrait && isCurationPromo
+                  ? styles.portraitImage
+                  : styles.image,
                 hasFixedAspectRatio
                   ? styles.imageFixedAspectRatio
                   : styles.imageResponsiveRatio,
+                isCurationPromo &&
+                  !isLoading && {
+                    [`@supports (filter: blur(15px))`]: {
+                      background: `rgba(${colour?.rgb?.join(',')}, 0.62)`,
+                    },
+                  },
+                isCurationPromo && {
+                  background: `rgb(${colour?.rgb?.join(',')})`,
+                },
```

Comment:
```suggestion
                isCurationPromo && {
                  background: `rgb(${colour?.rgb?.join(',')})`,
                  ...( !isLoading && {
                    [`@supports (filter: blur(15px))`]: {
                      background: `rgba(${colour?.rgb?.join(',')}, 0.62)`,
                    },
                  }),
```

Can we combine these? Wonder if there's a nicer way to handle all of these conditional styles...